constants/reforgestones.json

```diff
@@ -290,6 +290,30 @@
       }
     }
   },
+  "BOO_STONE": {
+    "internalName": "BOO_STONE",
+    "reforgeName": "Greater Spook",
+    "reforgeType": "blacksmith/reforge_stone",
+    "itemTypes": "ARMOR",
+    "requiredRarities": [
+      "EPIC",
+      "LEGENDARY"
+    ],
+    "reforgeCosts": {
+      "EPIC": 10000,
+      "LEGENDARY": 10000
+    },
+    "reforgeAbility": {
```

Comment:
You could just do `"reforgeAbility": "Grants §a+1 §5Fear§7.",` here.

Reply:
thx. And done.